packages/app/src/pages/_app.tsx

```diff
@@ -31,8 +31,8 @@ export default function App(props: AppProps) {
   const { Component, pageProps } = props;
 
   // const { locale = 'nl' } = useRouter(); // if we replace this with process.env.NEXT_PUBLIC_LOCALE, next export should still be possible?
-  const locale = process.env.NEXT_PUBLIC_LOCALE || 'nl';
-  const text = languages[locale as LanguageKey];
+  const locale = (process.env.NEXT_PUBLIC_LOCALE || 'nl') as LanguageKey;
+  const text = languages[locale];
```

Comment:
In hindsight it's better to do an assertion over here instead of inside the `useIntlHelperContext`.

 ```suggestion
  const locale = (process.env.NEXT_PUBLIC_LOCALE || 'nl') as LanguageKey;
  assert(['en', 'nl'].includes(locale), `Encountered unknown language: ${locale}`);
  
  const text = languages[locale];
```